package anote;


message Context {
  optional string id = 1;
  optional string title = 2;
  optional string description = 3;
}



message Project {
  optional string id = 1;
  optional string title = 2;
  optional string parent_id = 3;
  optional string description = 4;
  optional int64 created_at = 5;
  optional int64 modified_at = 6;
  optional int32 progress = 7;

  repeated Project sub_project = 8;
}
